docs/decisions/0001-open-analytics.rst

```diff
@@ -0,0 +1,95 @@
+1. Open Analytics
+##################################
+
+Status
+******
+
+**Draft** (-> Accepted)
+
+Context
+*******
+
+Analytics for Open edX has been neglected since Insights and the related repositories stopped being
+actively supported around 2018. The goal at the time was to eventually replace Insights, but
+that project was never picked up. While the community has done some recent work circa 2022 to update
+Insights, it was never a widely adopted solution for the wider community. Interviews with the community
+have revealed a few reasons why:
+
+- It is complicated, comprising 6 Github repositories, many pieces of infrastructure, and requiring
+  knowledge of several domain specific technologies to configure (Pandas, Hive, Sqoop, Hadoop,
+  Luigi, etc.)
+- It is expensive to run, and is in many ways specific to Amazon Web Services technologies
+- Turnaround time for data refreshes are on the order of a day or more in most cases
+- Documentation is out of date, further complicating any new adoption or alternative deployments
+- Discrepancies between Insights data calculations and data displayed in Studio have caused confusion
+
+Architectural decisions made post-Insights and new technologies have changed the analytics
+landscape, unlocking the ability to deliver analytical and operational data and display it in
+near-real time on commodity hardware with much simpler configuration and deployment. Additionally
+we have a wide variety of use cases in the Open edX community with differing requirements for
+privacy, scalability, budget, and expertise.
+
+Decision
+********
+
+We will create the Open Analytics Reference System (OARS) that combines existing open source projects
+into a preconfigured bundle that can be easily deployed using Tutor.
+
+This system will:
+
+- Transform existing Open edX tracking log events into an open standard format
+- Store them using a standards-compliant learning record store
+- Present a user interface of data visualizations secured via single-sign-on against the LMS
+- Allow download of report data for those with permissions to view it
+- Provide a secure API for integrations with other tools or data viewing methods
+
```

Comment:
I think PII deserves a mention here -- this reference system can provide a good example of how to display useful analytics using de-identified data.

Reply:
I've added some text around this, let me know what you think. I don't want to over-promise based on what I've seen come through in these events in the past.